acceptance/framework/k8s/debug.go

```diff
@@ -119,6 +119,17 @@ func WritePodsDebugInfoIfFailed(t *testing.T, kubectlOptions *k8s.KubectlOptions
 				writeResourceInfoToFile(t, replicaset.Name, "replicaset", testDebugDirectory, kubectlOptions)
 			}
 		}
+
+		// Describe any services.
+		services, err := client.CoreV1().Services(kubectlOptions.Namespace).List(context.Background(), metav1.ListOptions{LabelSelector: labelSelector})
```

Comment:
Do we want to do all namespaces here, or do we always deploy into the kubectlOptions.Namespace ns?

Reply:
this is the standard across the rest of the resource types.

Reply:
Sounds good!